astroquery/esa/hubble/tests/test_esa_hubble_remote.py

```diff
@@ -59,18 +59,24 @@ def test_query_tap_async(self):
     def test_download_product(self):
         result = esa_hubble.query_tap(query=self.hst_query)
         observation_id = np.random.choice((result['observation_id']))
-        temp_file = self.temp_folder.name + "/" + observation_id + ".tar"
+        temp_file = self.temp_folder.name + "/" + observation_id
```

Comment:
I'm afraid this won't work with windows (we still have a bit of a cleanup backlog for that, including some in the esa modules https://github.com/astropy/astroquery/issues/2773). So while we're at it, could you swap all these string manipulations with an `os.path.join` instead, here and below, too?

Reply:
Modified in different places so it works on every environment.